--- origsrc/djvulibre-3.5.25/config/acinclude.m4	2012-05-07 21:56:53.000000000 -0500
+++ src/djvulibre-3.5.25/config/acinclude.m4	2013-07-28 01:29:03.511464900 -0500
@@ -96,7 +96,6 @@ AC_DEFUN(AC_CXX_OPTIMIZE,[
    CFLAGS=
    for opt in $saved_CXXFLAGS ; do
      case $opt in
-       -g*) test $ac_debug != no && OPTS="$OPTS $opt" ;;
        -O*) ;;
        *) CXXFLAGS="$CXXFLAGS $opt" ;;
      esac
