--- origsrc/djvulibre-3.5.20/gui/nsdejavu/Makefile.in	2007-04-20 11:44:23.000000000 -0500
+++ src/djvulibre-3.5.20/gui/nsdejavu/Makefile.in	2008-05-11 13:32:11.875000000 -0500
@@ -27,7 +27,7 @@
 datadir = @datadir@
 libdir = @libdir@
 mandir = @mandir@
-plugindir = ${libdir}/netscape/plugins
+plugindir = ${libdir}/mozilla/plugins
 
 CC = @CC@
 CXX = @CXX@
@@ -61,7 +61,7 @@
 
 nsdejavu.la: nsdejavu.lo
 	${LIBTOOL} --mode=link ${CC} ${CFLAGS} nsdejavu.lo ${LIBS} \
-	  -rpath ${plugindir} -module -o $@
+	  -rpath ${plugindir} -module -avoid-version -o $@
 
 nsdejavu.x: nsdejavu.la FORCE
 	eval `grep '^dlname=' nsdejavu.la` && \
--- origsrc/djvulibre-3.5.20/i18n/de/Makefile.in	2007-04-20 11:44:23.000000000 -0500
+++ src/djvulibre-3.5.20/i18n/de/Makefile.in	2008-05-11 13:09:03.390625000 -0500
@@ -96,7 +96,7 @@
 .PHONY: FORCE
 .SUFFIXES: .ts .qm
 
-.ts.qm:
+%.qm: $(srcdir)/%.ts
 	if test ${LRELEASE} = ":" ; then : ; else \
 	  echo TRANSLATIONS=tmp.ts > tmp.pro && \
 	  cp $< tmp.ts && ${LRELEASE} tmp.pro && cp tmp.qm $@ && \
--- origsrc/djvulibre-3.5.20/i18n/fr/Makefile.in	2007-04-20 11:44:23.000000000 -0500
+++ src/djvulibre-3.5.20/i18n/fr/Makefile.in	2008-05-11 13:09:03.406250000 -0500
@@ -96,7 +96,7 @@
 .PHONY: FORCE
 .SUFFIXES: .ts .qm
 
-.ts.qm:
+%.qm: $(srcdir)/%.ts
 	if test ${LRELEASE} = ":" ; then : ; else \
 	  echo TRANSLATIONS=tmp.ts > tmp.pro && \
 	  cp $< tmp.ts && ${LRELEASE} tmp.pro && cp tmp.qm $@ && \
--- origsrc/djvulibre-3.5.20/i18n/ja/Makefile.in	2007-09-17 01:28:45.000000000 -0500
+++ src/djvulibre-3.5.20/i18n/ja/Makefile.in	2008-05-11 13:09:03.406250000 -0500
@@ -97,7 +97,7 @@
 .PHONY: FORCE
 .SUFFIXES: .ts .qm
 
-.ts.qm:
+%.qm: $(srcdir)/%.ts
 	if test ${LRELEASE} = ":" ; then : ; else \
 	  echo TRANSLATIONS=tmp.ts > tmp.pro && \
 	  cp $< tmp.ts && ${LRELEASE} tmp.pro && cp tmp.qm $@ && \
